WeakAuras/RegionTypes/Icon.lua

```diff
@@ -98,6 +106,75 @@ local properties = {
     setter = "SetGlowColor",
     type = "color"
   },
+  glowLines = {
+    display = { WeakAuras.newFeatureString .. L["Glow"], L["Lines & Particules"]},
```

Comment:
```suggestion
    display = { WeakAuras.newFeatureString .. L["Glow"], L["Lines & Particles"]},
```
Particules is not the English spelling.